contracts/zero-ex/tests/forked/SwapEthForERC20Test.t.sol

```diff
@@ -0,0 +1,199 @@
+                address(fqtData.buyToken),
+                sources.UniswapV2Router
+            );
+
+            /*//////////////////////////////////////////////////////////////
+                                    BridgeAdapter
+            //////////////////////////////////////////////////////////////*/
+            // Now let's set up a UniswapV2 fill
+            fqtData.bridgeOrders = new IBridgeAdapter.BridgeOrder[](1);
+            IBridgeAdapter.BridgeOrder memory order;
+            // The ID is shifted so we can concat <PROTOCOL><NAME>
+            // e.g <UniswapV2Protocol><UniswapV2>
+            // or  <UniswapV2Protocol><SushiSwap> for forks
+            order.source = bytes32(uint256(BridgeProtocols.UNISWAPV2) << 128);
+            // How much we want to fill on this order, which can be different to the total
+            // e.g 50/50 split this would be half
+            order.takerTokenAmount = 1e18;
+            // Set this low as the price of ETH/USDT can change
+            order.makerTokenAmount = amountOut;
+            // The data needed specifically for the source to fill,
+            // e.g for UniswapV2 it is the router contract and a path. See MixinUniswapV2
+            address[] memory uniPath = new address[](2);
+            uniPath[0] = address(tokens.WrappedNativeToken);
+            uniPath[1] = address(tokens.USDT);
+            order.bridgeData = abi.encode(address(sources.UniswapV2Router), uniPath);
+            fqtData.bridgeOrders[0] = order;
+            // Now encode the FQT data into the transformation
+            transformations[1].data = abi.encode(fqtData);
+
+            vm.deal(address(this), 1e18);
+            uint256 balanceETHBefore = address(this).balance;
+            uint256 balanceERC20Before = IERC20TokenV06(tokens.USDT).balanceOf(address(this));
+
+            IZeroEx(payable(addresses.exchangeProxy)).transformERC20{value: 1e18}(
+                // input token
+                IERC20TokenV06(LibERC20Transformer.ETH_TOKEN_ADDRESS),
+                // output token
+                IERC20TokenV06(address(tokens.USDT)),
+                // input token amount
+                1e18,
+                // min output token amount
+                order.makerTokenAmount,
+                // list of transform
+                transformations
+            );
+
+            log_named_uint("        NativeAsset balance before", balanceETHBefore);
+            log_named_uint("        ERC-20 balance before", balanceERC20Before);
+            log_named_uint("        NativeAsset balance after", balanceETHBefore - address(this).balance);
+            log_named_uint(
+                "        ERC-20 balance after",
+                IERC20TokenV06(tokens.USDT).balanceOf(address(this)) - balanceERC20Before
+            );
+            assert(IERC20TokenV06(tokens.USDT).balanceOf(address(this)) > 0);
+        } else {
+            log_string("    Liquidity Source Not available on this chain");
```

Comment:
Is this really a soft error? Seems like a missing liquidity source that's as basic as uniswap would be a significant test failure